src/business_logic/bus_stop/include/IBus_stop.hpp

```diff
@@ -22,6 +22,7 @@ namespace cpp_playground::pg_business_logic::bus_stop
         virtual void draw() = 0;
         virtual int counterBussesStop() = 0;
```

Comment:
Consider verbs as more meaningful name for methods, e.g. getVehiclesCount()

```suggestion
        virtual int getVehiclesCount() = 0;
```